src/tenderly/tenderly.config.ts

```diff
@@ -51,7 +51,7 @@ export const TENDERLY_CHAINS: number[] = [
   ChainId.RSK, // RSK - 30
   ChainId.WCC, // WorldChain - 480
   ChainId.LSK, // Lisk - 1135
-  ChainId.SON, // Sonic - 146
+  // ChainId.SON, // Sonic - 146
```

Comment:
Sonic is pending now, since the new lifi/types does not include this chain, we gonna remove it